.git-blame-ignore-revs

```diff
@@ -15,4 +15,6 @@ f125c2664d3bffb59536f4d3dfee7fa5323c7721
 32e21b64138876a065081ab4bbe15b1fbea12316
 8ab847ca4cbe30baeb0e5d1cbd68ba40385b01ae
 077012c5bc43649fc705698859143226e48686cf
+# Converting to file scoped namespaces on 3/26/2025. 
+077012c5bc43649fc705698859143226e48686cf
```

Comment:
will update to the right commit once https://github.com/dotnet/roslyn/pull/77854 goes in.